index.html

```diff
@@ -0,0 +1,191 @@
+      <header class="header contaniner">
+        <nav class="nav">
+          <div class="nav__logo">
+            <img src="/images/logo.svg" alt="" />
+          </div>
+          <ul class="nav__list">
+            <li class="nav__link">
+              <a class="nav__link__a" href="#team">The Team</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__link__a" href="#pricing">Pricing</a>
+            </li>
+            <li class="nav__link">
+              <a class="nav__link__a" href="#features">Features</a>
+            </li>
+          </ul>
+        </nav>
+        <section class="hero">
+          <div class="hero__text">
+            <h1 class="hero__h1">
+              Beautiful Free Nova<br />
+              template
+            </h1>
+            <p class="hero__p">
+              A top notch premium quality <br />template for your next web<br />
+              project.
+            </p>
+            <button class=" btn btn--hero">Download FREE!</button>
+          </div>
+          <div class="hero__image">
+            <img
+              class="hero__img"
+              src="/images/screen.png"
+              alt=""
+            />
+          </div>
+        </section>
+      </header>
+    </section>
+    <section id="features" class="section__background--two ">
+      <div class="contaniner">
+        <h1 class="sectiontwo__h1">Features</h1>
+        <div class="features">
+            <div class="feature">
+            <div class="bcg__image"> <img
+              class="feature__img"
+              src="/images/responsive.svg"
+              alt=""
+            /></div>
+            <h3 class="feature__h3">Fully responsive</h3>
+            <p class="feature__p">
+              Looks amazing on any device:<br> smartphone, tablet,laptop and<br> desktop
+            </p>
+          </div>
+          <div class="feature">
+            <div class="bcg__image"> <img
+              class="feature__img"
+              src="/images/customizable.svg"
+              alt=""
+            /></div>
+            <h3 class="feature__h3">customizable</h3>
+            <p class="feature__p">
+              Change the colors, pictures or <br> any of the sections to suit your <br>needs.
+            </p>
+          </div>
+          <div class="feature">
+            <div class="bcg__image"> <img
+              class="feature__img"
+              src="/images/design.svg"
+              alt=""
+            /></div>
+            <h3 class="feature__h3">slick and beautiful <br>design</h3>
+            <p class="feature__p">
+              Trendy and fresh design,fits<br> any website 
+            </p>
+          </div> 
+      </div>
+    </section>
+    <section class="section__background--three container">
+       <img class="img__tools" src="/images/logos.png" alt="">
+    </section>
+    <section class ="section__background--four">
+        <div class="section__four container--max">
```

Comment:
Jeśli mamy modyfikator to powinien występować też blok czyli `section__four container container--max`.
Jesli to nie pasuje tutaj to znaczy, że źle użyliśmy tego modyfikatora - raczej powinno być `section__four section__four--max`